### PR TITLE
Frontend is up and running

### PR DESCRIPTION
**Pull request by NSBPittman**
User can create account, login, and when they have a token get their id (note this is just a demo to show that GETs that rely on JWT work)

### CODE REVIEW

File: src/main/webapp/app.js [hunk 1]
```diff
@@ -8,19 +8,15 @@ appModule.controller('MainCtrl', ['mainService','$scope','$http',
         $scope.greeting = 'BookBuddaroos';
         $scope.token = null;
         $scope.error = null;
-        $scope.header = null;
-//            $scope.roleUser = false;
```
Comment (dlesnef1) [inline src/main/webapp/app.js:12]: less commented out code, wooooo

**Comment by ShelbyCohen** [comment]
Looks great because nick is great!

File: src/main/webapp/app.js [hunk 2]
```diff
@@ -109,44 +98,17 @@ appModule.service('mainService', function($http) {
...
-//            return $http.post('http://localhost:8080/users', data).then(function(response){
-//                return response.data.token;
-//            })
         },
 
-        getAccountDetails : function(header) {
+        getAccountDetails : function() {
             var data = "Authorization	: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJOaWNrIiwiaWQiOjF9.KhStwKp6-ma3ZxYI8EhLD8oRHz8AVnWNJC37-QljOMc'";
 
-//            var settings = {
```
Comment (dlesnef1) [inline src/main/webapp/app.js:120]: Yum yum copy pasta yum yum

File: src/main/webapp/app.js (hunk 1 above)
Comment (NSBPittman) [inline src/main/webapp/app.js:12]: I think commented out code provides an aesthetic we are now missing

File: src/main/webapp/app.js (hunk 2 above)
Comment (NSBPittman) [inline src/main/webapp/app.js:120]: copy pasta is a staple in my diet. OKAY DAVE?!?!

**Pull request merged**